### PR TITLE
UI elements

### PR DESCRIPTION
**Pull request by animeshgarai09**
Auth and profile page design completed. PR is only for design elements.

Auth page : [https://felix-social-react-git-ui-elements-animeshgarai09.vercel.app/](https://felix-social-react-git-ui-elements-animeshgarai09.vercel.app/)
profile page: [https://felix-social-react-git-ui-elements-animeshgarai09.vercel.app/profile](https://felix-social-react-git-ui-elements-animeshgarai09.vercel.app/profile)

### CODE REVIEW

File: src/components/sidenav/sidenav.module.scss [hunk 1]
```diff
@@ -77,9 +79,23 @@
         }
     }
 }
+.options {
+    position: absolute !important;
+    right: 1rem;
+    top: 1rem;
+    z-index: 10;
+    font-size: 1.6rem !important;
+    color: var(--clr-gray-2) !important;
```
Comment (nayyyhaa) [inline src/components/sidenav/sidenav.module.scss:95]: Suggestion: Try not to use !important in CSS.

File: src/hooks/useInputHandler.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { useState } from "react"
+
+const useInputHandler = (state) => {
+    const [inputState, setInputState] = useState(state)
+    const inputChange = (evt) => {
+        const type = evt.target.type;
+        const value = evt.target.value;
+        const name = evt.target.name
+        console.log(type)
```
Comment (nayyyhaa) [inline src/hooks/useInputHandler.js:9]: can remove console.log from here.

File: src/pages/authentication/sub-components/signin.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        setSubmitState(true)
+        // handleSignIn(inputState.email, inputState.password, from, setSubmitState)
+    }
+
+    const handleGuest = () => {
+        setGuestState(true)
+        // handleSignIn("animeshgarai09@gmail.com", "testing1234", from, setGuestState)
```
Comment (nayyyhaa) [inline src/pages/authentication/sub-components/signin.js:36]: can remove commented code if not required.

**Review by nayyyhaa — APPROVED** [review]
Your UI looks awesome as always!! Nice work Animesh, loved that you used scss for styling + all the components are well-written & reusable. 💯

**Review by hsnice16 — APPROVED** [review]
Great work, man. The UI is superb. I think the other reviewee has suggested all the changes. There is nothing new to suggest from my side. 
Good Job.

**Pull request merged**